dotcom-rendering/src/experiments/tests/sign-in-gate-time-of-day.ts

```diff
@@ -0,0 +1,23 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const signInGateTimesOfDay: ABTest = {
+	id: 'SignInGateTimesOfDay',
+	start: '2023-12-11',
+	expiry: '2023-12-15',
```

Comment:
Putting next friday as the expiry for buffer. I'll release a revert PR on Thusday